astropy/modeling/core.py

```diff
@@ -899,6 +899,12 @@ def bounding_box(self, limits):
     def evaluate(self, *args, **kwargs):
         """Evaluate the model on some input variables."""
 
+    def render(self, *args, **kwargs):
+        """
```

Comment:
@adonath - This discussion happened early in #3909. https://github.com/astropy/astropy/pull/3909#discussion_r33900230
https://github.com/astropy/astropy/pull/3909#discussion_r33952549
Further thoughts on this @cdeil and @embray?